docs/_data/menu-sql.yaml

```diff
@@ -123,37 +123,27 @@
             - text: SELECT 
               url: sql-ref-syntax-qry-select.html 
               subitems:
-                - text: DISTINCT Clause
-                  url: sql-ref-syntax-qry-select-distinct.html
-                - text: Joins
-                  url: sql-ref-syntax-qry-select-join.html
-                - text: ORDER BY Clause 
-                  url: sql-ref-syntax-qry-select-orderby.html
+                - text: WHERE Clause 
```

Comment:
The order of these clauses depends on something? Can we reorder them along with the main menu? https://github.com/apache/spark/blob/master/docs/sql-ref-syntax-qry-select.md

Reply:
@maropu Did you want me to re-order menu in the left hand side panel as per the "related clauses" in https://github.com/apache/spark/blob/master/docs/sql-ref-syntax-qry-select.md ? i.e 
```
- [WHERE Clause](sql-ref-syntax-qry-select-where.html)
- [GROUP BY Clause](sql-ref-syntax-qry-select-groupby.html)
- [HAVING Clause](sql-ref-syntax-qry-select-having.html)
- [ORDER BY Clause](sql-ref-syntax-qry-select-orderby.html)
- [SORT BY Clause](sql-ref-syntax-qry-select-sortby.html)
- [CLUSTER BY Clause](sql-ref-syntax-qry-select-clusterby.html)
- [DISTRIBUTE BY Clause](sql-ref-syntax-qry-select-distribute-by.html)
- [LIMIT Clause](sql-ref-syntax-qry-select-limit.html)
```

Reply:
Yea, can we?

Reply:
@maropu Sure... will do.

Reply:
@maropu I have reconciled the link order. Also i have added a very brief description in the entry page which looks like this : 
<img width="972" alt="Screen Shot 2020-01-27 at 10 35 17 PM" src="https://user-images.githubusercontent.com/14225158/73240922-527a6500-4155-11ea-94e3-dc84bc7e642b.png">

Please see if this looks okay to you or if we want to re-word or add anything ?

cc @huaxingao @srowen